client/reader/following-stream/index.jsx

```diff
@@ -97,6 +106,38 @@ module.exports = React.createClass( {
 				this.showSelectedPost( { replaceHistory: true } );
 			}
 		}
+
```

Comment:
@blowery Though the problem seems to be outside the scope of this PR, I prefer auto updating too. How about creating a new issue and having discussions there?